style.css

```diff
@@ -0,0 +1,202 @@
+@import url("https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Jost&display=swap");
+
+/* Body */
+body {
+  background-color: #f0f0f0;
+  margin-left: 40px;
+  margin-right: 40px;
+}
+
+/* Header */
+header {
+  padding: 20px;
+  background-color: #f0f0f0;
+}
+
+h1 {
+  font-family: "Bebas Neue", sans-serif;
+  font-size: 50px;
+  color: #0018a4;
+  letter-spacing: 1px;
+}
+
+@media (max-width: 768px) {
+  h1 {
+    font-size: 35px;
+  }
+}
+
+.filter,
+.sort,
+.random {
+  display: inline-block;
+  margin-right: 60px;
+}
+
+.filter h2,
+.sort h2,
+.random h2 {
+  font-family: "Jost", sans-serif;
+  font-size: 18px;
+  letter-spacing: 0.5px;
+  margin-bottom: 10px;
+}
+
+@media (max-width: 768px) {
+  .filter h2,
+  .sort h2,
+  .random h2 {
+    font-size: 14px;
+  }
+}
+
+/* Buttons within header */
+.buttons {
+  display: flex;
+  justify-content: center;
+  margin-left: -5px;
+}

```

Comment:
all media tags should live at the bottom of the CSS file as it will be compiled from top to bottom. The rule that is set last, is the one that will be executed, so it's important to position them correctly not to get any rules accidentally overwritten 👍